resources/benchmark-runner.mjs

```diff
@@ -90,19 +90,18 @@ class PageElement {
         this._node.focus()
     }
 
-    change(options = NATIVE_OPTIONS)
+    dispatchEvent(eventName, options = NATIVE_OPTIONS)
     {
-        this._node.dispatchEvent(new Event('change', options))
-    }
-
-    input(options = NATIVE_OPTIONS)
-    {
-        this._node.dispatchEvent(new Event('input', options))
+        if (eventName == 'submit') {
+            // FIXME FireFox doesn't like `new Event('submit')
+            this._dispatchSubmitEvent();
+        }  else {
+            this._node.dispatchEvent(new Event(eventName, options))
+        }
     }
 
-    submit(options = NATIVE_OPTIONS)
+    _dispatchSubmitEvent()
     {
-        // FIXME FireFox doesn't like `new Event('submit')
```

Comment:
Looks like this comment about Firefox is no longer applicable.
We should probably just use dispatchEvent instead.

Reply:
Which version are you using? FF 107.0 (64-bit) still get's stuck on AngularJS-TodoMVC if I don't use the custom event.

Reply:
That is surprising when the c++ code for new Event literally calls initEvent
https://searchfox.org/mozilla-central/rev/670e2e0999f04dc7734c8c12b2c3d420a1e31f12/dom/events/Event.cpp#365-367

Reply:
> That is surprising when the c++ code for new Event literally calls initEvent https://searchfox.org/mozilla-central/rev/670e2e0999f04dc7734c8c12b2c3d420a1e31f12/dom/events/Event.cpp#365-367

I agree it's surprising, though I observe the same behavior on Nightly.